string plugin_name
string library_path
---
string error_msg

